xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/PropertyClass.java

```diff
@@ -210,6 +213,11 @@ public void displayHidden(StringBuffer buffer, String name, String prefix, BaseC
     @Override
     public void displayView(StringBuffer buffer, String name, String prefix, BaseCollection object,
         XWikiContext context)
+    {
+        displayViewNoEscape(buffer, name, object);
+    }
+
+    protected void displayViewNoEscape(StringBuffer buffer, String name, BaseCollection object)
```

Comment:
new API, missing unstable annotation too. I'm not fond of adding these 2 new APIs.

Reply:
Don't use StringBuffer, StringBuilder is recommended.